doc: create_devstack base domain argument (#63)
appsembler/devstack: README.rst

.. code:: bash

    $ make amc-shell  # Inside your machine to open the AMC shell
    $ ./manage.py create_devstack_site blue localhost # Inside the AMC shell
    $ make lms-shell  # Inside your machine to open the LMS shell
    $ ./manage.py lms create_devstack_site blue localhost

> **Note:** Sultan users can replace the base domain ``localhost`` in ``create_devstack_site`` commands with something else like `devstack`.

Reviewer: Since we are specifically mentioning Sultan users, let's make it more obvious for them:

```suggestion
**Note:** Sultan users can replace the base domain ``localhost`` in ``create_devstack_site`` commands with any host specified in their Sultan `EDX_HOST_NAMES` configuration.
```

Author: @ahmedaljazzar that's better. I like it. But what's the `EDX_HOST_NAMES`?

Reviewer: @OmarIthawi It's a configuration variable in `Sultan` allows user to specify as many hosts as they want. https://github.com/appsembler/sultan/blob/master/configs/.configs#L78
